src/js/components/search/visualizations/time/TimeVisualizationSection.jsx

```diff
@@ -49,6 +59,64 @@ export default class TimeVisualizationSection extends React.Component {
         }
     }
 
+    downloadLabel = () => {
+        let periodLabel;
+        if (this.props.data.visualizationPeriod === 'fiscal_year') {
+            periodLabel = 'Year';
+        }
+        else {
+            periodLabel = capitalize(this.props.data.visualizationPeriod);
+        }
+        return `Download data by ${periodLabel}`;
+    };
+
+    downloadData = () => {
```

Comment:
can we rename this to describe what's being returned?

like `get...?`

Reply:
What would you suggest? What's being returned is the data for the download. True, function names should be verbs, and this doesn't actually produce the download...

Reply:
Sorry, missed your last line here...

Reply:
done